packages/fluent/scss/panelbar/_layout.scss

```
@@ -0,0 +1,151 @@
+@use "_variables.scss" as *;
+@use "../core/color-system" as *;
+@use "../core/variables" as *;
+@use "../icon/variables" as *;
+
+@mixin kendo-panelbar--layout() {
+
+    .k-panelbar {
+        margin: 0;
+        padding: 0;
+        border-width: var( --kendo-panelbar-border-width, #{$kendo-panelbar-border-width} );
+        border-style: var( --kendo-panelbar-border-style, #{$kendo-panelbar-border-style} );
+        box-sizing: border-box;
+        outline: 0;
+        font-family: var( --kendo-panelbar-font-family, #{$kendo-panelbar-font-family} );
+        font-size: var( --kendo-panelbar-font-size, #{$kendo-panelbar-font-size} );
+        line-height: var( --kendo-panelbar-line-height, #{$kendo-panelbar-line-height} );
+        list-style: none;
+        display: block;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
+        *,
+        *::before,
+        *::after {
+            box-sizing: border-box;
+        }
+
+
+        // Root
+        > .k-item,
+        > .k-panelbar-header {
+            // TODO
+            border-width: 0;
+            border-style: var( --kendo-panelbar-item-border-style, #{$kendo-panelbar-item-border-style} );
+            border-color: inherit;
+            display: block;
+
+            > .k-link {
+                padding-inline: var( --kendo-panelbar-header-padding-x-start, #{$kendo-panelbar-header-padding-x-start} ) var( --kendo-panelbar-header-padding-x-end, #{$kendo-panelbar-header-padding-x-end} );
```

we can split this for better readability:

```suggestion
            padding-inline-start: var( --kendo-panelbar-header-padding-x-start, #{$kendo-panelbar-header-padding-x-start} );
            padding-inline-end: var( --kendo-panelbar-header-padding-x-end, #{$kendo-panelbar-header-padding-x-end} );
```